core/src/main/java/io/questdb/PropertyKey.java

```diff
@@ -447,6 +447,11 @@ public static Optional<PropertyKey> getByString(String name) {
         return Optional.ofNullable(nameMapping.get(name));
     }
 
+    @Override
+    public String getEnvVarEquivalent() {
+        return "QDB_" + propertyPath.replace('.', '_').toUpperCase();
```

Comment:
nit: might be a good idea to move this to default implementation on the interface to avoid having to update `EntPropertyKey`.

Reply:
good idea, done